config/api.yml

```diff
@@ -859,6 +859,28 @@
         :identifier: miq_template_protect
       - :name: resolve
         :identifier: miq_template_policy_sim
+  :tenants:
+    :description: Tenants
+    :options:
+    - :collection
+    :methods: *70174834084700
+    :klass: Tenant
+    :collection_actions:
+      :post:
+      - :name: create
+        :identifier: rbac_tenant_add
```

Comment:
add the edit and delete in the collection post, supports multiple tenant edit/delete.